main.go

```diff
@@ -1,7 +1,46 @@
 package main
 
-import "fmt"
+import (
+	"net/http"
+	"os"
+
+	"github.com/Gamebuildr/Dave/client"
+	"github.com/Gamebuildr/Dave/pkg/config"
+	"github.com/Gamebuildr/Dave/pkg/scaler"
+)
 
 func main() {
-	fmt.Printf("Hello from Dave")
+	cli := client.DaveClient{}
+	cli.Create()
+	scaleMrRobot(&cli)
+	scaleGogeta(&cli)
+	http.ListenAndServe(":3000", nil)
+}
+
+func scaleMrRobot(cli *client.DaveClient) {
+	mrrobotScaler := scaler.HTTPScaler{
+		LoadAPIUrl:    "/api/endpoint",
+		AddLoadAPIUrl: "/api/endpoint",
+		Client:        &http.Client{},
+	}
+	system := scaler.ScalableSystem{
+		System:  mrrobotScaler,
+		MaxLoad: 3,
```

Comment:
If we add it to an environment it might be a bit harder - we could create tiered levels though or something. And yea it can be changed on the fly - its just overriding the structs value (as long as you have the reference to the original struct).